CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 # Changelog
 
+## v1.22.0
+
+This version introduces an agent setting to select the method of
+translating SNMP objects. The agent setting "snmp_translator" can be
+"netsnmp" which translates by calling external programs snmptranslate
+and snmptable, or "gosmi" which translates using the built-in gosmi
+library.
```

Comment:
No need to do hard line-breaks. It isn't done in the rest of this document. The suggestion also adds some markup
```suggestion
This version introduces an agent setting to select the method of translating SNMP objects.
The agent setting **snmp_translator** can be *netsnmp* which translates by calling external programs `snmptranslate` and `snmptable`, or *gosmi* which translates using the built-in gosmi library.
```